src/workloads/replication/startup/1_0_5GB.yml

```diff
@@ -0,0 +1,36 @@
+SchemaVersion: 2018-07-01
```

Comment:
Can you add full documentation like [this](https://github.com/mongodb/genny/blob/c69842cfa7ddbc2d81966be344bdb9357cba5dc7/src/workloads/execution/BackgroundIndexConstruction.yml#L3-L17).

The description needs have to talk about what this is for, how it works, why we want it and what metrics are important. 

The _Keywords:_ field is also important.

Reply:
I thought the `README` was enough, but I will add documentation everywhere